tests/sentry/incidents/endpoints/test_organization_alert_rule_details.py

```diff
@@ -172,7 +172,7 @@ def test_with_unresponsive_sentryapp(self):
         with self.feature("organizations:incidents"):
             resp = self.get_response(self.organization.slug, self.rule.id)
 
-        assert len(responses.calls) == 1
+        # assert len(responses.calls) == 1
```

Comment:
Should we just delete this now, or is there a better assertion we could be making here?